src/Beacon.jsx

```diff
@@ -220,7 +254,7 @@ export class Beacon extends Component {
       }
     }
 
-    const tooltipClass = tooltipActive ? 'tour-in' : !this.state.tooltipHidden && 'tour-out';
+    const tooltipClass = tooltipActive ? 'tour-in' : (!this.state.tooltipHidden && 'tour-out') || 'inactive';
```

Comment:
Here we use that new `inactive` style to hide the tooltip after the user displays and then dismisses it.